galaxykit/namespaces.py

```diff
@@ -103,10 +104,30 @@ def delete_v1_namespace(client, name):
     return client.delete(delete_url, parse_json=False)
 
 
-def delete_namespace(client, name):
+def delete_namespace(client, name, recurse=False):
```

Comment:
isn´t `cascade` a better name for this argument? following what we see in DB relations

Reply:
sure